ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java

```diff
@@ -573,7 +573,7 @@ private KsqlPlan sourceTablePlan(
     final MutableMetaStore tempMetastore = new MetaStoreImpl(new InternalFunctionRegistry());
     final Formats formats = ddlCommand.getFormats();
     tempMetastore.putSource(new KsqlTable<>(
-        statement.getStatementText(),
+        statement.getUnMaskedStatementText(),
```

Comment:
This still needs to be updated.

Reply:
Not sure if we mask too much here. If 
```
    "mode"='bulk',
    "topic.prefix"='jdbc-',
    "table.whitelist"='users',
    "key"='username');
```
are all masked, is there another way for user to actually check it?

Reply:
Connectors are not stored in the metastore, so this is not an issue for connectors. And this only matters for valid statements, so CREATE STREAM/TABLE should not be masked.

If in the future we introduce new statement types which also need to be masked, then it will be up to whoever introduces the new type of statement to mask the statement in a way that the important info is still available to users.